api/Navigator.json

```diff
@@ -253,17 +253,6 @@
                 "version_added": "11",
                 "version_removed": "16",
                 "prefix": "moz"
-              },
-              {
-                "version_added": "10",
-                "version_removed": "16",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "dom.battery.enabled",
```

Comment:
I'm deleting this entry entirely in https://github.com/mdn/browser-compat-data/pull/9626.